src/messaging/messaging-types.ts

```diff
@@ -718,36 +699,41 @@ function validateAndroidNotification(notification: AndroidNotification) {
   }
 
   if (typeof notification.eventTimestamp !== 'undefined') {
-    let zuluTimestamp: string;
+    if (!(notification.eventTimestamp instanceof Date)) {
+      throw new FirebaseMessagingError(
+        MessagingClientErrorCode.INVALID_PAYLOAD, 'android.notification.eventTimestamp must be a valid `Date` object');
+    }
     // Convert timestamp to RFC3339 UTC "Zulu" format, example "2014-10-02T15:01:23.045123456Z"
-    zuluTimestamp = notification.eventTimestamp.toISOString();
+    const zuluTimestamp: string = notification.eventTimestamp.toISOString();
```

Comment:
Nit: You don't have to add type annotations for single-line initializers like this. But it's fine to leave them as is too.

Reply:
Thanks!